### PR TITLE
Demo Pull Request

### PR DESCRIPTION
**Pull request by smith-kyle**

### CODE REVIEW

File: code-example.py [hunk 1]
```diff
@@ -14,14 +14,19 @@ def divide(a, b):
     if b != 0:
```
Comment (smith-kyle) [inline code-example.py:14]: _Commented on notebook `code-example.py` Cell 1 Line 17_
```python
    return a * b


def divide(a, b):
    if b != 0:
        return a / b
    else:
        raise ValueError("Division by zero is not allowed")
```
raise a value error

File: pandas-example.ipynb [hunk 2]
```diff
@@ -11,24 +11,24 @@
       "cell_type": "markdown",
```
Comment (smith-kyle) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 10 From line 1 to 3_
```python
rng = np.random.RandomState(42)
A = pd.DataFrame(rng.randint(0, 20, (2, 2)),
                 columns=list('AB'))
```
Comment posted in GitNotebooks

**Review by smith-kyle — COMMENTED** [review]
My review

File: pandas-example.ipynb (hunk 2 above)
Comment (smith-kyle) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 4 From line 1 to 2_
```python
import pandas as pd
import numpy as np
```
Example comment posted in GItNotebooks

Comment (smith-kyle) [reply pandas-example.ipynb:11]: Example reply posted in GitHub

Comment (smith-kyle) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 10 From line 1 to 3_
```python
rng = np.random.RandomState(42)
A = pd.DataFrame(rng.randint(0, 20, (2, 2)),
                 columns=list('AB'))
```
my example comment

Comment (smith-kyle) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 12 Line 1_
```markdown
## Ufuncs: Operations Between DataFrame and Series
```
My markdown comment

**Review by smith-kyle — COMMENTED** [review]
My example review

File: pandas-example.ipynb (hunk 2 above)
Comment (smith-kyle) [reply pandas-example.ipynb:11]: add reply comment in github

File: code-example.py (hunk 1 above)
Comment (smith-kyle) [inline code-example.py:14]: _Commented on notebook `code-example.py` Cell 1 From line 20 to 24_
```python
def modulus(a, b):
    return a % b


def main():
```
My code comment

File: sklearn-example.ipynb [hunk 3]
```diff
@@ -16,7 +16,7 @@
     },
```
Comment (smith-kyle) [inline sklearn-example.ipynb:16]: _Commented on notebook `sklearn-example.ipynb` Cell 7 Line 1_
```markdown
### Unsupervised learning: Dimensionality reduction
```
My markdown comment

Comment (smith-kyle) [inline sklearn-example.ipynb:16]: _Commented on notebook `sklearn-example.ipynb` Cell 3 Line 2_
```python
import seaborn as sns
iris = sns.load_dataset('iris')
```
another notebook code comment

**Review by smith-kyle — COMMENTED** [review]
My review

File: sklearn-example.ipynb (hunk 3 above)
Comment (smith-kyle) [reply sklearn-example.ipynb:16]: my reply in github

Comment (smith-kyle) [inline sklearn-example.ipynb:16]: _Commented on notebook `sklearn-example.ipynb` Cell 5 From line 1 to 2_
```python
iris['PCA1'] = X_2D[:, 0]
iris['PCA2'] = X_2D[:, 1]
```
My code comment

Comment (smith-kyle) [inline sklearn-example.ipynb:16]: _Commented on notebook `sklearn-example.ipynb` Cell 7 Line 1_
```markdown
### Unsupervised learning: Dimensionality reduction
```
My markdown

**Review by smith-kyle — COMMENTED** [review]
My review

File: pandas-example.ipynb (hunk 2 above)
Comment (smith-kyle) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 11 From line 18 to 19_
```python
fill = A.stack().sum()
A.add(B, fill_value=fill)
```
Adding comment to fill =

Comment (smith-kyle) [reply pandas-example.ipynb:11]: Add a reply in GitHub

Comment (smith-kyle) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 8 Line 1_
```python
A.subtract(B, fill_value=0)
```
Comment 1

Comment (smith-kyle) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 3 Line 4_
```markdown
One of the essential pieces of NumPy is the ability to perform quick element-wise operations, both with basic arithmetic (addition, subtraction, multiplication, etc.) and with more sophisticated operations (trigonometric functions, exponential and logarithmic functions, etc.).
Pandas inherits much of this functionality from NumPy, and the ufuncs that we introduced in [Computation on NumPy Arrays: Universal Functions](https://gitnotebooks.com/blog) are key to this.

Pandas includes a couple useful twists, however: for unary operations like negation and trigonometric functions, these ufuncs will *preserve index and column labels* in the output, and for binary operations such as addition and multiplication, Pandas will automatically *align indices* when passing the objects to the ufunc.
```
Markdown comment

**Review by smith-kyle — COMMENTED** [review]
my Neutral review

File: pandas-example.ipynb (hunk 2 above)
Comment (owen-connor) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 5 Line 2_
```markdown
Any item for which one or the other does not have an entry is marked with ``NaN``, or "Not a Number," which is how Pandas marks missing data (see further discussion of missing data in [Handling Missing Data](03.04-Missing-Values.ipynb)).
This index matching is implemented this way for any of Python's built-in arithmetic expressions; any missing values are filled in with NaN by default:
```
LGTM

Comment (smith-kyle) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 1 Line 1_
```markdown
# Operating on Data in Pandas
```
test

Comment (smith-kyle) [reply pandas-example.ipynb:11]: my reply

Comment (smith-kyle) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 11 Line 18_
```python
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff

fill = A.stack().sum()
```
test

Comment (smith-kyle) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 11 Line 18_
```python
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff

fill = A.stack().sum()
```
test

Comment (smith-kyle) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 2 Line 1_
```markdown
Copied from [https://github.com/jakevdp/PythonDataScienceHandbook](https://github.com/jakevdp/PythonDataScienceHandbook)
```
test

Comment (smith-kyle) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 11 Line 18_
```python
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff

fill = A.stack().sum()
```
my comment

Comment (smith-kyle) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 11 Line 18_
```python
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff

fill = A.stack().sum()
```
my comment

Comment (smith-kyle) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 11 Line 18_
```python
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff

fill = A.stack().sum()
```
test

Comment (smith-kyle) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 11 From line 24 to 27_
```python
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
```
here's a comment

Comment (owen-connor) [reply pandas-example.ipynb:11]: Reply

Comment (owen-connor) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 9 Line 1_
```markdown
Observe that the indices align accurately regardless of their sequence in the two objects, and the result's indices are organized in ascending order.
```
Comment

Comment (owen-connor) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 11 Line 24_
```python

fill = A.stack().sum()
A.add(B, fill_value=fill)

# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
# Large cells? No problem. Cells are collapsed to showcase the diff
```
Another Comment

**Review by owen-connor — APPROVED** [review]
Approved.

File: code-example.py (hunk 1 above)
Comment (smith-kyle) [inline code-example.py:14]: _Commented on notebook `code-example.py` Cell 1 From line 17 to 20_
```python
        raise ValueError("Division by zero is not allowed")


def modulus(a, b):
```
here's a comment on the code example

File: pandas-example.ipynb (hunk 2 above)
Comment (smith-kyle) [inline pandas-example.ipynb:11]: _Commented on notebook `pandas-example.ipynb` Cell 4 From line 1 to 2_
```python
import pandas as pd
import numpy as np
```
wat

**Pull request closed without merging**